dev/buildtool/ga-defaults.yml

```diff
@@ -35,6 +35,7 @@
 # Commented values surrounded by '[]' are code defaults.
 # Not all code defaults are shown, only those for commenting purposes.
 
+# root_path:
```

Comment:
Extraneous, or just there for documentation?

Reply:
docs